### PR TITLE
fix: linting, license dates, and import statements in react

### PR DESCRIPTION
**Pull request by evenstensberg**
Leaving as it is

### CODE REVIEW

**Review by rmmayo — CHANGES_REQUESTED** [review]
hi @evenstensberg - thanks for another contribution!  After looking over the changes, we *think* that the copyright years for existing files are supposed to retain the year that they were created (from reading [this SO question](https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated)), so we would like to change the `Copyright 2021 SkillTree` changes back to `Copyright 2020 SkillTree`, and also [this](https://github.com/NationalSecurityAgency/skills-client/pull/110/files) change to `es2021` seems to have broken the build.  The other changes to `skills-client-react` look good - thank you!

File: skills-client-integration/skills-int-client-ng/tsconfig.base.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "moduleResolution": "node",
     "importHelpers": true,
     "target": "es2015",
-    "module": "es2020",
+    "module": "es2021",
```
Comment (rmmayo) [inline skills-client-integration/skills-int-client-ng/tsconfig.base.json:14]: @evenstensberg  this change seems to have broken the build.

**Comment by evenstensberg** [comment]
I'll open another PR!

**Comment by evenstensberg** [comment]
There we go, PTAL

**Review by rmmayo — APPROVED** [review]
Looks good, thanks!  👍

**Comment by evenstensberg** [comment]
Do you have any other open issues I can help with?

**Comment by sudo-may** [comment]
@evenstensberg awesome!! Feel free to grab [#567](https://github.com/NationalSecurityAgency/skills-service/issues/567) and then anything labeled with "[good first issue](https://github.com/NationalSecurityAgency/skills-service/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)" would make sense as well; you will probably want to also take a look at the [Development Environment](https://code.nsa.gov/skills-docs/contribution/devEnv.html#prerequisites) portion of the docs.

**Comment by sudo-may** [comment]
@evenstensberg on a second thought [#129](https://github.com/NationalSecurityAgency/skills-service/issues/129) maybe a lower hanging fruit

**Comment by evenstensberg** [comment]
Thx! I'll get back to you once I have time!

**Pull request merged**